scripts/test.sh

```diff
@@ -6,7 +6,7 @@ if [ -n "$1" ]; then
     export CODACY_PROJECT_TOKEN="$1"
 fi
 
-sbt coverage test
+sbt coverage +test
```

Comment:
We don't usually test multiple scala versions. I'm not sure on how this will work with the coverage report. @lolgab